app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -28,7 +28,10 @@ function fetchNext(el, endpoint, insertCallback) {
         nextPage += 1;
         insertCallback(entries);
         if (entries.length === 0) {
-          document.getElementById('loading-articles').style.display = 'none';
+          const loadingElement = document.getElementById('loading-articles');
```

Comment:
Refactor (non-blocking): You can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

Reply:
I don't think I can in the vanilla JS code here (I initially tried 😆 )